pages/map.js

```diff
@@ -86,7 +86,10 @@ export class Map extends Component {
               <Body>
                 <p>
                   {t("map.contact_1") + " "}
-                  <AnchorLink fontSize={18} href={"tel:+" + t("contact.phone")}>
+                  <AnchorLink
+                    fontSize="18px"
```

Comment:
I think fontSize needs to be rewritten like this in guided_experience_summary.js. It's currently `fontSize={24}` in 2 places